.browserslistrc

```diff
@@ -0,0 +1,2 @@
+defaults
+not ie 11
```

Comment:
@dhruvkb Included this since most of the repos break when run on ie 11.
Also let me know other browsers with the versions on which the website break.